application/values-test.yaml

```
@@ -1307,7 +1307,7 @@ cronJob:
             cpu: 1
 
 # Take backup of application namespace
-backup:
+schedule:
```

Instead of `schedule` we can be a little more explicit about what this is for in the value's own name.

```suggestion
backupSchedule:
```

What do you think, @AsfaMumtaz?